torch/_dynamo/decorators.py

```diff
@@ -47,9 +53,17 @@ def run(fn=None):
     return RunOnlyContext()
 
 
+def _old_compile_disable_behavior():
+    return config.old_compile_disable_behavior or not justknobs_check(
+        "pytorch/compiler:compile_enable_disable"
```

Comment:
you don't have to fix it now, but this is a pretty bad JK name; why doesn't it have "old" or "behavior" in its name? Is it enabling something or disabling something?

Reply:
How are you testing the alternate JK setting :think:

Reply:
Added a `test_old_compile_disable` test that sets `config.old_compile_disable_behavior` to true.